templates/create-daml-app/package.json

```diff
@@ -0,0 +1,28 @@
+{
+    "scripts": {
+        "test": "yarn workspace create-daml-app run test"
+    },
+    "private": true,
+    "workspaces": [
+        "daml-ts/057eed1fd48c238491b8ea06b9b5bf85a5d4c9275dd3f6183e0e6b01730cc2ba",
```

Comment:
These hashes look like they are going to break everytime the hash of daml-stdlib and daml-prim changes. Can we avoid that? I thought daml2ts can generate this file?

Reply:
Yes you're right, I think we can not check in this file (or just not include the generated workspaces) and daml2ts will fill in the gaps. I'm just confirming that behaviour

Reply:
daml2ts on head does not generate separate workspaces for packages anymore so this goes away